imperial_coldfront_plugin/views.py

```diff
@@ -109,13 +109,12 @@ def send_group_invite(request: HttpRequest) -> HttpResponse:
             invite_url = request.build_absolute_uri(
                 reverse("imperial_coldfront_plugin:accept_group_invite", args=[token])
             )
-
-            # Send invitation via email.
-            send_mail(
-                "You've been invited to a group",
-                f"Click the following link to accept the invitation:\n{invite_url}",
-                request.user.email,
+            send_email_in_background(
                 [invitee_email],
+                "HPC Access Invitation",
+                "You've been invited to join the access group of "
+                f"{request.user.get_full_name()}\n\n"
```

Comment:
Maybe add the username or email in brackets. As one of many James Turners, I know it can be annoying confusing to be called on name alone.

Reply:
As a Cave-Ayland this is not a problem I ever experience.

Reply:
Done